lib/main.dart

```diff
@@ -9,13 +9,16 @@ export 'package:easy_localization_loader/src/json_asset_loader.dart';
 import 'package:useful_useless_app/src/core/provider/user_provider.dart';
 import 'package:useful_useless_app/ui/login_screen.dart';
 
+import 'src/core/provider/calendar_scroll_provider.dart';
+
 Future<void> main() async {
   WidgetsFlutterBinding.ensureInitialized();
   await Firebase.initializeApp();
   runApp(MultiProvider(
     providers: [
       ChangeNotifierProvider(create: (_) => UserProvider()),
       ChangeNotifierProvider(create: (_) => PowerOffProvider()),
+      ChangeNotifierProvider(create: (_) => CalendarScrollProvider()),
```

Comment:
we don't need this provider so high in the tree. wrap the widget that uses it with this provider